src/common/logger/LoggerUtil.ts

```
@@ -0,0 +1,78 @@
+            user: user,
+            timestamp: new Date().toISOString(),
+        });
+    }
+
+    static error(
+        message: string,
+        error?: string,
+        context?: string,
+        user?: string,
+    ) {
+        this.getLogger().error({
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)